### PR TITLE
Issue #2375 Fixing the sprint planning script for CDK milestones

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: scripts/planning/sprint-planning.sh [hunk 1]
```diff
@@ -52,7 +52,7 @@ function github_milestone()
 
 function jira_milestone()
 {
-  milestone_data=`curl -s -X GET -H "Content-Type: application/json" "https://issues.jboss.org/rest/api/2/search?jql=project=CDK%20AND%20component=minishift%20AND%20fixVersion=${milestone}"`
+  milestone_data=`curl -s -X GET -H "Content-Type: application/json" "https://issues.jboss.org/rest/api/2/search?jql=project=CDK%20AND%20fixVersion=${milestone}"`
```
Comment (praveenkumar) [inline scripts/planning/sprint-planning.sh]: @LalatenduMohanty I am not sure if I am doing something wrong but not able to run that curl command on my mac/Linux box

```
$ curl -s -X GET -H "Content-Type: application/json" "https://issues.jboss.org/rest/api/2/search?jql=project=CDK%20AND%20fixVersion=${milestone}"
{"errorMessages":["Error in JQL Query: Expecting either a value, list or function before the end of the query."],"errors":{}}
```

Comment (praveenkumar) [reply scripts/planning/sprint-planning.sh]: nvm, I forgot to put milestone and now able to get the list.

File: scripts/planning/sprint-planning.sh [hunk 2]
```diff
@@ -52,7 +52,7 @@ function github_milestone()
 
 function jira_milestone()
 {
-  milestone_data=`curl -s -X GET -H "Content-Type: application/json" "https://issues.jboss.org/rest/api/2/search?jql=project=CDK%20AND%20component=minishift%20AND%20fixVersion=${milestone}"`
```
Comment (praveenkumar) [inline scripts/planning/sprint-planning.sh]: So we just now removed the minishift from the component and only care about the CDK and the given milestone :+1:

Comment (LalatenduMohanty) [reply scripts/planning/sprint-planning.sh]: @praveenkumar yes because we few components other than `minishift`.

**Pull request merged**